cohorts/load.py

```diff
@@ -1037,3 +1124,33 @@ def filter_not_null(df, col):
     if updated_len < original_len:
         print("Missing %s for %d patients: from %d to %d" % (col, original_len - updated_len, original_len, updated_len))
     return df
+
+def _provenance_str(provenance):
```

Comment:
This is now a duplicate of `provenance_str` in `load_from_cache`, right? Can we get rid of that one?